api/time_weighted_averages.md

```diff
@@ -0,0 +1,144 @@
+# Time-weighted average functions
+The functions related to time-weighted averages.
+
+## time_weight() (point form)
+
+```SQL
+time_weight(
+    method TEXT,
+    ts TIMESTAMPTZ,
+    value DOUBLE PRECISION
+) RETURNS TimeWeightSummary
+```
+
+Only two values for `method` are currently supported: `linear` and `LOCF`, and
```

Comment:
I think we lost some asterisks/footnote markings here?

Reply:
No, I left the interpolation methods content in the GH, and added an xref. I think there's probably an argument for having a separate section for it in the docs, though, we refer to it a lot.

Reply:
It's possible it's also missing in ours, but I think that the method bit had an asterisk in the function ie:
```
time_weight
     method TEXT**,
```
** Only two values...
which to me made it more clear, but might not work with some formatting things?

Reply:
It should work.